en/docs/develop/customizations/customizing-the-developer-portal/enabling-or-disabling-self-signup.md

```diff
@@ -66,4 +66,4 @@ In order to disable self signup for a particular tenant, follow the steps below.
     ![Self registration disabled](../../../assets/img/learn/self-signup-disabled.png)
 
 !!! tip
-    To engage your own signup process, see [Adding a User Signup Workflow](/learn/consume-api/customizations/adding-a-user-signup-workflow).
+    To engage your own signup process, see [Adding a User Signup Workflow](https://apim.docs.wso2.com/en/latest/develop/customizations/adding-a-user-signup-workflow/).
```

Comment:
@akurathimohan, thank you for your contribution! Please note that the path should be added as {{base_path}}/develop/customizations/adding-a-user-signup-workflow/ so that the path will not have to be changed with each and every APIM doc version. I updated your PR